pysd/py_backend/vensim/vensim2py.py

```diff
@@ -886,12 +958,19 @@ def visit_call(self, n, vc):
             # remove dimensions info (produced by !)
 
             function_name = vc[0].lower()
-            arguments = [e.strip() for e in vc[4].split(",")]
+            arguments = []
+            while len(','.join(arguments)) < len(vc[4]):
+                arguments.append(self.args.pop())
+                arguments = [arguments[-1]] + arguments[:-1]
```

Comment:
When I merged #223, I removed these lines as they were not commented and not specific tests were added in the cited PR.
However, the conflict you are trying to solve with these is already solved with PR #249  where the parser is cleaned and reading arguments as `param` has been removed.

Reply:
You are right. My error, I had not seen the issue #248 solved before I made this pull request.